doc/specs/vulkan/chapters/pipelines.txt

```diff
@@ -771,6 +771,18 @@ endif::VK_KHR_maintenance2[]
     The number of resources in pname:layout accessible to each shader stage
     that is used by the pipeline must: be less than or equal to
     sname:VkPhysicalDeviceLimits::pname:maxPerStageResources
+ifdef::VK_NV_clip_space_w_scaling[]
+  * If no element of the pname:pDynamicStates member of pname:pDynamicState
+    is ename:VK_DYNAMIC_STATE_VIEWPORT_W_SCALING_NV, and the
+    pname:viewportWScalingEnable, member of a
```

Comment:
nit: extraneous comma

Reply:
It is not (I think), but the sentence structure is a mess.
Lemme scale it back for ya:
> [...] and the pname:viewportWScalingEnable, [blah blah], is `VK_TRUE` [then blah blah]